jonprl-mode.el

```diff
@@ -242,8 +330,8 @@ This list is constructed from JonPRL's output.")
       (message "Press 'q' to close the development.")
       (let ((inhibit-read-only t))
         (erase-buffer)
-        (call-process jonprl-path nil t t file-name))
-      (goto-char (point-min)))))
+        (call-process jonprl-path nil t t command-args))
+      )))
```

Comment:
Lisp convention is to put the closing parens on the line before, with what they close. So
```
       (call-process jonprl-path nil t t command-args)))))
```